### ISSUE TITLE
Campaign does not load/detect any contact in respective segment.

### ISSUE THREAD

**Issue by rikkot** [mautic/mautic#7759]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
Having loaded contacts and created a Contact Segment I tried creating a test email campaign and send it to this new segment added in the Campaign Builder.

**Campaign Setup**
![Campaign Flow](https://user-images.githubusercontent.com/53613292/62371055-74ea4780-b551-11e9-8385-c37f367611d5.jpg)

**Contact Segment displays number of contacts.**
![Contact Segment](https://user-images.githubusercontent.com/53613292/62415118-51eb9080-b642-11e9-9da6-9691880ca0e3.jpg)

**Contact not loaded**
![Mautic Contact not loaded](https://user-images.githubusercontent.com/53613292/62371007-508e6b00-b551-11e9-9312-f2ab20ab0f9c.jpg)

| Q   | A
| --- | ---
| Mautic version | 2.15.1
| PHP version | 7.1.30
| Browser | Chrome

### Steps to reproduce
1. Created test Campaign and Email
2. Campaign does not load Contact Segments.
3. Already tried Cron Job Commands and Terminals
_Commands Used_:
php /path/to/mautic/app/console mautic:segments:update
_Result_:
Rebuilding contacts for segment 2
0 total contact(s) to be added in batches of 300
0 total contact(s) to be removed in batches of 300
0 contact(s) affected

Rebuilding contacts for segment 6
0 total contact(s) to be added in batches of 300
0 total contact(s) to be removed in batches of 300
0 contact(s) affected

Rebuilding contacts for segment 3
0 total contact(s) to be added in batches of 300
0 total contact(s) to be removed in batches of 300
0 contact(s) affected

Rebuilding contacts for segment 4
0 total contact(s) to be added in batches of 300
0 total contact(s) to be removed in batches of 300
0 contact(s) affected

php /path/to/mautic/app/console mautic:campaigns:rebuild
_Result_:
Nothing Displayed
 

### Log errors
Non Displayed

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by escopecz** [comment]
Aren't you missing the campaigns:update command? See https://www.mautic.org/docs/en/setup/getting_started.html

**Comment by rikkot** [comment]
> Aren't you missing the campaigns:update command? See https://www.mautic.org/docs/en/setup/getting_started.html

Ohh I did do that as well. Missed out on mentioning it here.

![terminal-a](https://user-images.githubusercontent.com/53613292/62553690-6f606a80-b88d-11e9-82e2-841740c4041b.jpg)

I also tried re installing the whole thing again and still stuck at the same spot. 
My test campaigns not populating contacts from the selected segment.

**Comment by escopecz** [comment]
That's what the `mautic:campaigns:update` does. It should also output all the campaigns that it goes through similar to the segment update command. This is what it does for me:
```
$ app/console mautic:campaigns:update
Rebuilding contacts for campaign 1
0 total contact(s) to be added in batches of 300
0 total contact(s) to be removed in batches of 300
...
```
Do you have the campaign published?

**Comment by rikkot** [comment]
> at's what the `mautic:campaigns:update` does. It should also output all the campaigns that it goes through similar to the segment update command. This is what it does for me:

True I also thought it should display something after I enter the commands.

And no the campaign is unpublished at the moment. I'll try publishing it and then trying these commands again.

**Comment by escopecz** [comment]
What if you try to update one specific campaign?
`$ app/console mautic:campaigns:update -i 99`
(replace 99 with the actual ID of your campaign) 
Still no output?

**Comment by rikkot** [comment]
> What if you try to update one specific campaign?
> `$ app/console mautic:campaigns:update -i 99`
> (replace 99 with the actual ID of your campaign)
> Still no output?

No output

![campaign update command](https://user-images.githubusercontent.com/53613292/62616855-201e4680-b92e-11e9-88f9-7ba4f8297ff6.jpg)

**Comment by rikkot** [comment]
> @rikkot were you able to solve this? What was the problem?

I was missing a step. Like you suggested I had to publish the Campaign for the contact to be populated via the Segment. Latter it started to pause, stating not enough resources. All I had to do was to pause the Import (on import page on my Mautic) -> refresh the page -> start the Import again and refresh page -> re enter import command via Terminal. I feel even using a Cron Job would help to fix this but I was in a bit of a hurry so used Terminal instead to see it happen live.
